draft-privacypass-rate-limit-tokens.md

```diff
@@ -120,14 +121,113 @@ periodically, say, once per month. For example, an origin may serve ten (major
+limit policy to apply. This rate limit policy is returned to the attester (according
+to {{response-one}}), which then checks whether or not the client is within this
+policy. If yes, the attester forwards the issuer token response to the client (according
+to {{response-two}}) so that the resulting token can be redeemed to the origin.
+
+~~~
+                                                     +-----------+
+      Client                                         |   Origin  |
+        |                    Challenge               |           |
+        <-------------------------------------------------+      |
+        |                                            |           |
+        |          +-------------+                   |           |
+        |          |   Attester  |                   |           |
+        |          |             |                   |           |
+        |          | Attest      |    +----------+   |           |
+        +----------------->      |    |  Issuer  |   |           |
+        |          |             |    |          |   |           |
+        |          |       TokenRequest          |   |           |
+        |          |      [ + encrypted origin ] |   |           |
+        +---------------------------------->     |   |           |
+        |          |             |    |          |   |           |
+        |          |             |    |          |   |           |
+        |          |       TokenResponse         |   |           |
+        |          |      [ + rate limit ]       |   |           |
+        |          |     <-------------------+   |   |           |
+        |          |             |    |          |   |           |
+        |          |  in limit?  |    |          |   |           |
+        |          |     yes     |    |          |   |           |
+        |          |             |    |          |   |           |
+        |   TokenResponse        |    |          |   |           |
+        <-----------------+      |    |          |   |           |
+        |          |             |    |          |   |           |
+        |          +-------------+    +----------+   |           |
+        |                                            |           |
+        |                    Redeem                  |           |
+        +------------------------------------------------->      |
+                                                     |           |
+                                                     +-----------+
+~~~
+{: #fig-example-allow title="Successful rate-limited issuance"}
+
+{{fig-example-deny}} shows how this interaction works for client requests that
+exceed the rate limit. The client's request to the issuer and the issuer's
+response to the attester are the same. However, in this scenario, the client
+is not within the rate limit, so the attester responds to the client with an
+error instead of the issuer's token response.
```

Comment:
```suggestion
{{fig-example-deny}} shows how this interaction works for client requests that
exceed the rate limit. The Client's request to the Issuer and the Issuer's
response to the Attester are the same. However, in this scenario, the Client
is not within the rate limit, so the Attester responds to the Client with an
error instead of the issuer's token response.
```